cpp/include/cuvs/neighbors/common.hpp

```diff
@@ -209,7 +280,7 @@ auto make_aligned_dataset(const raft::resources& res, const SrcT& src, uint32_t
 }
 
 /** Parameters for VPQ compression. */
-using vpq_params = cuvs::neighbors::ann::vpq_params;
+using vpq_params = cuvs::neighbors::vpq_params;
```

Comment:
Does not look like this line is needed anymore as the class definition is in the same file and same namespace now

Reply:
Good catch!